### PR TITLE
Add AMT

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Adds parameters and variables from [`AMT`](https://github1s.com/pslmodels/tax-calculator/blob/HEAD/taxcalc/calcfunctions.py#L1859) in `taxcalc`.

### CODE REVIEW

File: openfisca_us/parameters/irs/income/amt/brackets.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    period: year
+  1:
+    values:
+      2013-01-01: 179_500.0
+      2014-01-01: 182_500.0
+      2015-01-01: 185_400.0
+      2016-01-01: 186_300.0
+      2017-01-01: 187_800.0
+      2018-01-01: 191_100.0
+      2019-01-01: 194_800.0
```
Comment (MaxGhenis) [inline openfisca_us/parameters/irs/income/amt/brackets.yaml:23]: newline

File: openfisca_us/variables/irs/inputs.py [hunk 2]
```diff
@@ -831,6 +841,16 @@ class e24515(Variable):
     documentation = """Sch D: Un-Recaptured Section 1250 Gain"""
 
 
+class filer_e24515(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = u"Section 1250 Gain (un-recaptured) for the tax unit (excluding dependents)"
+    definition_period = YEAR
```
Comment (MaxGhenis) [inline openfisca_us/variables/irs/inputs.py:848]: should this and others also have `unit = "currency_USD"`?

File: openfisca_us/variables/irs/outputs.py [hunk 3]
```diff
@@ -754,8 +758,121 @@ class c09600(Variable):
...
+        line54 = min_(line48, line53)
+        cgtax2 = line54 * cg.rates["2"]
+        line56 = line47 + line54
+        line57 = where(line41 == line56, 0, line46 - line56)
+        linex2 = where(line41 == line56, 0, max_(0, line54 - line48))
+        cgtax3 = line57 * cg.rates["3"]
+        line61 = where(
+            line38 == 0,
+            0,
+            0.25 * max_(0, (line30 - line41 - line56 - line57 - linex2)),
```
Comment (MaxGhenis) [inline openfisca_us/variables/irs/outputs.py:851]: See https://github.com/PSLmodels/Tax-Calculator/issues/2635

**Review by MaxGhenis — COMMENTED** [review]
Haven't dug into the logic yet

File: openfisca_us/variables/irs/inputs.py (hunk 2 above)
Comment (nikhilwoodruff) [reply openfisca_us/variables/irs/inputs.py:850]: Yep, added in the rest of the changed variables

File: openfisca_us/parameters/irs/income/amt/brackets.yaml (hunk 1 above)
Comment (nikhilwoodruff) [reply openfisca_us/parameters/irs/income/amt/brackets.yaml]: Added

File: openfisca_us/variables/irs/outputs.py [hunk 4]
```diff
@@ -754,8 +758,121 @@ class c09600(Variable):
...
+            tax_unit(variable, period)
+            for variable in [
+                "dwks10",
+                "dwks13",
+                "dwks14",
+                "dwks19",
+                "filer_e24515",
+            ]
+        ]
+        form_6251_part_iii_requred = np.any(
```
[the remark this answers is not in the document]
Comment (nikhilwoodruff) [reply openfisca_us/variables/irs/outputs.py]: Fixed

File: openfisca_us/variables/irs/outputs.py [hunk 5]
```diff
@@ -754,8 +758,121 @@ class c09600(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
+    label = "Alternative Minimum Tax"
+    unit = "currency-GBP"
```
[the remark this answers is not in the document]
Comment (nikhilwoodruff) [reply openfisca_us/variables/irs/outputs.py]: Fixed

**Comment by nikhilwoodruff** [comment]
@MaxGhenis - so it turns out the OpenFisca logic was correct here: because the calcfunctions in taxcalc are not independent of each other, my approach of generating a unit test by feeding the relevant final variables into the function was incorrect. I've added the script generating unit tests, plus ten random (from the CPS, with non-zero outputs) unit tests for AMT.

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**